backend/danswer/utils/gpu_utils.py

```diff
@@ -0,0 +1,28 @@
+import requests
+
+from danswer.utils.logger import setup_logger
+from shared_configs.configs import MODEL_SERVER_HOST
+from shared_configs.configs import MODEL_SERVER_PORT
+
+logger = setup_logger()
+
+
+def gpu_status_request() -> bool:
+    model_server_url = f"{MODEL_SERVER_HOST}:{MODEL_SERVER_PORT}"
```

Comment:
We will want this to have an argument for indexing: bool or something. We want to check the indexing model server for the multipass indexing.